src/components/Uploader/Uploader.md

```diff
@@ -0,0 +1,22 @@
+Uploader that accepts multiple files example:
+
```

Comment:
We have some different ways to use the uploader component as is shown in the design specs. It would be good to see those examples here too.

Also, a better and more detailed description would be great for a new user accessing our lib

Reply:
@welitonderesende this component has a detailed description, but it's not located in this file. It is being done through JSDoc. Check out the [deployment preview](https://herzui-67km44ybh-iomicromed.vercel.app/#uploader). Do you think it's ok?

Can you clarify to me what are the different ways to use the uploader that you mean? I'm not sure of any other way that is not being used